### DISCUSSION TITLE
.NET blog post for WPF?

### DISCUSSION THREAD

**Discussion by bgrainger** [dotnet/wpf#7147] [General]
Will there be a .NET blog post about changes in WPF for .NET 7?

I really liked the [Performance Improvements in .NET 7](https://devblogs.microsoft.com/dotnet/performance_improvements_in_net_7/) blog post, particularly how community contributors were featured and credited.

Now that WPF has started accepting community contributions, a blog post coinciding with the .NET 7 release seems like a great way to feature the work of @lindexi, @ThomasGoulet73, @batzen and others (including a couple of contributions from me, let's be honest 😀). It would also be nice to mention the (many) performance improvements from @stephentoub.

**Comment by pchaurasia14** [comment]
@bgrainger - Absolutely yes. Will publish it soon 👍

**Reply by bgrainger** [reply]
Just saw the post; thanks!

https://devblogs.microsoft.com/dotnet/wpf-on-dotnet-7/
